src/project-css/app/Adapters/ImportAdapter.php

```diff
@@ -0,0 +1,208 @@
+<?php
+/**
+ * @author Tracy A McCormick <tam0013@mail.wvu.edu>
+ */
+
+namespace App\Adapters;
+
+use Illuminate\Support\Facades\DB;
+use App\Models\Collection;
+use App\Models\Table;
+use App\Helpers\CSVHelper;
+use App\Helpers\CustomStringHelper;
+
+use Log;
+
+class ImportAdapter {
+    public $savedTkns;
+    public $lastErrRow;
+    public $tkns;
+    public $curArry;
+    public $clmnLst;
+    public $tblNme;
+
+     /**
+     * Takes 2 arrays of tokens and merges them.
+     * Designed around the Rockefeller data their was instances where a
+     * incorrect character caused a break in reading the line. When this 
+     * is detected due to a inconsistent field count we will attempt to 
+     * merge the 2 lines.
+     * @param array $tkns1
+     * @param array $tkns2
+     * @return array
+     */    
+    public function mergeLines() {
+        $numItem = count($this->savedTkns) - 1;
+        $this->savedTkns[ $numItem ] = $this->savedTkns[ $numItem ] . ' ' . $this->tkns[ 0 ];
+        unset($this->tkns[ 0 ]);
+        $this->tkns = ( (count($this->tkns) > 0) ? array_merge($this->savedTkns, $this->tkns) : $this->savedTkns );
+    }
+
+    /**
+     * takes a string and prepares it to be used inserted as a new record
+     * if we do not find enough items in the line we will check and see if
+     * the previous line was saved and merge them and check the count again.
+     * if their is insufficent items we will save the tkns and the row position
+     * so we can attempt a merge later.
+     * @param string $curLine current line read from the file
+     * @param string $delimiter type of delmiter that is used in the file
+     * @param integer $orgCount current number of fields expected
+     * @param integer $prcssd current position in the file
+     * @return string
+     */
+    public function prepareLine($curLine, $delimiter, $orgCount, $prcssd) {
+        // Strip out Quotes that are sometimes seen in csv files around each item
+        $curLine = str_replace('"', "", $curLine);
```

Comment:
So your saying sometimes the data is setup as 

"item1", "items2", "some other really cool item here...", "another item" ?

Reply:
The reason I'm asking that is that I would say if you have ever had an issue with special characters this is the area systemic of creating the error.  Valid CSV data that has special characters are always wrapped in quotes as a way of escaping the data.